src/library/trackcollection.cpp

```diff
@@ -158,8 +158,8 @@ bool TrackCollection::unhideTracks(const QList<TrackId>& trackIds) {
     // Emit signal(s)
     // TODO(XXX): Emit signals here instead of from DAOs
     QSet<CrateId> modifiedCrateSummaries(
-            m_crates.collectCrateIdsOfTracks(trackIds));
-    emit(crateSummaryChanged(modifiedCrateSummaries));
+      m_crates.tracks().collectCrateIdsOfTracks(trackIds));
+    //emit(m_crates.crateSummaryChanged(modifiedCrateSummaries));
```

Comment:
Commenting out code without good reason (that is explained by a comment) is bad practice.